instances/technique/anterogradeTracing.jsonld

```diff
@@ -4,11 +4,11 @@
   },
   "@id": "https://openminds.ebrains.eu/instances/technique/anterogradeTracing",
   "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
-  "definition": null,
-  "description": null,
+  "definition": "Anterograde tracing is a technique used to trace axonal projections from their source (the cell body or soma) to their point of termination (the synapse).",
+  "description": "Anterograde tracing is a technique used to trace axonal projections from their source (the cell body or soma) to their point of termination (the synapse). Anterograde tracers are taken up by neuronal cell bodies at the injection site and travel to the axon terminals. Anterograde tracing techniques allow for a detailed assessment of neuronal connections between a target population of neurons and their outputs throughout the nervous system.",
```

Comment:
I would typically not reuse the definition again in the description to avoid redundancy for the reader. Maybe reformulate or delete and switching the order of the remaining description sentences?